.eslintrc.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
 module.exports = {
-    extends: '@swellaby/eslint-config/lib/bundles/ts-node'
+    extends: '@swellaby/eslint-config/lib/bundles/ts-node',
+    'rules': {
+        'no-unused-vars': [
+            'error',
+            {
+                argsIgnorePattern: '^_'
```

Comment:
As an FYI I've opened https://github.com/swellaby/eslint-config/issues/8, as we should really have this baked in by default. TSLint and the main linters for other languages recognize the underscore syntax (`_req` for example) by default automatically, but apparently allows you to define your own ignore pattern.